packages/pds/src/api/app/bsky/graph/subscribeMuteList.ts

```diff
@@ -0,0 +1,17 @@
+import { Server } from '../../../../lexicon'
+import AppContext from '../../../../context'
+
+export default function (server: Server, ctx: AppContext) {
+  server.app.bsky.graph.subscribeMuteList({
+    auth: ctx.accessVerifier,
+    handler: async ({ auth, input }) => {
+      const { list } = input.body
+      const requester = auth.credentials.did
+
+      await ctx.services.account(ctx.db).subscribeMuteList({
+        list,
```

Comment:
Do we want any special behavior if the users tries to use a weird `list` at-uri?  E.g. if they attempt to use a post as a list, or something nonsensical like that.

Reply:
yeah can't hurt to give some extra user feedback 👍